bin/daemon

```diff
@@ -29,17 +29,26 @@ end
 @checksum = calc_checksum
 
 module Updater
+  class UpdateFailed < StandardError
```

Comment:
`class UpdateFailed < StandardError; end`

I think latest ruby 2.3 revisions add support for skipping the `;` :3
https://github.com/mruby/mruby/commit/2574adab48d0995204ee45b812125b2134c6c8f3